lessons/lesson3.md

```diff
@@ -0,0 +1,13 @@
+## Lesson 3: End to end test
+In the previous lessons, we have setup two firefly apps:
+- Event provider that automatically generating events 
+- Event consumer that automatically pulling from journaling api and write to firefly storage
+
+Let’s first set in `manifest.yml` and try the `/whisk.system/alarms/interval` feed of the OpenWhisk Alarm Package to fire trigger events on an interval base schedule. To see the effect instantly, we will make it run every minute. You will need a trigger set up with the `/whisk.system/alarms/interval` feed, and a rule to wire this trigger to the `publish-event` or `consume-event` action created earlier.
```

Comment:
Mhh I don't understand this part ? What are you supposed to do in `manifest` ?

Reply:
if people would like to use the alarm package to automatically generate event and pulling event